core-meta.html

```diff
@@ -87,20 +79,29 @@
          * @type String
          * @default ''
          */
-        key: String
-      },
-
-      bind: {
-        value: 'valueChanged',
-        key: 'keyChanged'
-      },
+        key: {
+          type: String,
+          value: '',
+          notify: true,
+          observer: 'keyChanged'
+        },
 
-      configure: function () {
-        return {
-          type: 'default',
-          key: '',
-          value: this
-        };
+        /**
+         * The value of the meta-data to store. If no value is specified, the
+         * value defualts to a self reference (`this`).
+         *
+         * @attribute value
+         * @type *
+         * @default this
+         */
+        value: {
+          type: String,
+          value: function () {
+            return this;
+          },
+          notify: true,
```

Comment:
I think it should have been in there for the first port. But that's mostly speculation on my part. Seems like something implementors would be interested in.